apisix/plugins/http-logger.lua

```diff
@@ -45,7 +45,8 @@ local schema = {
             }
         },
         concat_method = {type = "string", default = "json",
-                         enum = {"json", "new_line"}}
+                         enum = {"json", "new_line"}},
+        ssl_verify = {type = "boolean", default = true},
```

Comment:
For compatibility, we have to use false by default.

Reply:
ok